<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="load_tab">
     <attribute name="title">
      <string>Load</string>
     </attribute>
     <widget class="QPushButton" name="loadButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="scale_tab">
     <attribute name="title">
      <string>Scale</string>
     </attribute>
     <widget class="QLabel" name="kx_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>kx:</string>
      </property>
     </widget>
     <widget class="QLabel" name="ky_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ky:</string>
      </property>
     </widget>
     <widget class="QLabel" name="kz_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>kz:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="kx_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ky_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="kz_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="scale_button">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rotate_tab">
     <attribute name="title">
      <string>Rotate</string>
     </attribute>
     <widget class="QLabel" name="angle_x_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>70</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>angle of x:</string>
      </property>
     </widget>
     <widget class="QLabel" name="angle_y_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>70</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>angle of y:</string>
      </property>
     </widget>
     <widget class="QLabel" name="angle_z_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>70</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>angle of z:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="angle_x_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="angle_y_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="angle_z_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rotate_button">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Rotate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="transfer_tab">
     <attribute name="title">
      <string>Transfer</string>
     </attribute>
     <widget class="QLabel" name="dx_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>dx:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dy_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>dy:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dz_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>dz:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dx_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dy_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dz_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="transfer_button">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Transfer</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>21</y>
      <width>491</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
